### ISSUE TITLE
pytorch1.0 instead of 0.4.0

### ISSUE THREAD

**Issue by shen1994** [laughtervv/DepthAwareCNN#27]
*how to build `make.sh` in pytorch1.0, when load model in c++, it needs to convert model to script.Could you give me some reference?Thanks

**Comment by E18301194** [comment]
@shen1994  I met the same problem as you. Have you solved it?  thx

**Comment by shen1994** [comment]
solved, you need to read a little code about at.h   same api in it 843224306 邮箱：843224306@qq.com 签名由 网易邮箱大师 定制 On 12/07/2019 22:13, E18301194 wrote: @shen1994 I met the same problem as you. Have you solved it? thx — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by lqzhao** [comment]
> solved, you need to read a little code about at.h   same api in it 843224306 邮箱：843224306@qq.com 签名由 网易邮箱大师 定制 On 12/07/2019 22:13, E18301194 wrote: @shen1994 I met the same problem as you. Have you solved it? thx — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

Hi, how did you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!

**Comment by E18301194** [comment]
Have you reproduced the accuracy of the paper?
Although I can run through the code, the accuracy of the paper has been far behind





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "laughtervv/DepthAwareCNN"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年6月23日(星期二) 下午3:46
收件人:&nbsp;"laughtervv/DepthAwareCNN"<DepthAwareCNN@noreply.github.com&gt;;
抄送:&nbsp;"金刚狼的胡子"<949517097@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [laughtervv/DepthAwareCNN] pytorch1.0 instead of 0.4.0 (#27)





  
solved, you need to read a little code about at.h &nbsp; same api in it 843224306 邮箱：843224306@qq.com 签名由 网易邮箱大师 定制 On 12/07/2019 22:13, E18301194 wrote: @shen1994 I met the same problem as you. Have you solved it? thx — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.
  
Hi, how did you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!

**Comment by shen1994** [comment]
> @shen1994 I met the same problem as you. Have you solved it? thx

see DepthAwareStereo

**Comment by shen1994** [comment]
> id you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!

see DepthAwareStereo

**Comment by shen1994** [comment]
>  of the paper has been far behin

see DepthAwareStereo

**Comment by shen1994** [comment]
> > solved, you need to read a little code about at.h   same api in it 843224306 邮箱：[843224306@qq.com](mailto:843224306@qq.com) 签名由 网易邮箱大师 定制 On 12/07/2019 22:13, E18301194 wrote: @shen1994 I met the same problem as you. Have you solved it? thx — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> Hi, how did you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!

see https://github.com/shen1994/DepthAwareStereo

**Comment by Looottch** [comment]
> > id you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!
> 
> see DepthAwareStereo

Your code can't be used in pytorch1.6 right? Cause I try to build ops depthconv from your github page but I got error like "No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda' "

**Comment by shen1994** [comment]
> > > id you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!
> > 
> > 
> > see DepthAwareStereo
> 
> Your code can't be used in pytorch1.6 right? Cause I try to build ops depthconv from your github page but I got error like "No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda' "

see testing， training is the same with origin, but tesing has been changed much

**Comment by Looottch** [comment]
> > > > id you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!
> > > 
> > > 
> > > see DepthAwareStereo
> > 
> > 
> > Your code can't be used in pytorch1.6 right? Cause I try to build ops depthconv from your github page but I got error like "No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda' "
> 
> see testing， training is the same with origin, but tesing has been changed much
I checked testing codes in folder DepthAwareStereo/testing/models/ops/depthconv/src and it's actual original code. build.py is the only different file, will it work when you only change ffi to cpp_extension in build file?

**Comment by shen1994** [comment]
> only different file, will it

yes, cpp_extension will support pytorch1.0+, I have tested these codes in real scene.

**Comment by shen1994** [comment]
> > > > > id you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!
> > > > 
> > > > 
> > > > see DepthAwareStereo
> > > 
> > > 
> > > Your code can't be used in pytorch1.6 right? Cause I try to build ops depthconv from your github page but I got error like "No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda' "
> > 
> > 
> > see testing， training is the same with origin, but tesing has been changed much
> > I checked testing codes in folder DepthAwareStereo/testing/models/ops/depthconv/src and it's actual original code. build.py is the only different file, will it work when you only change ffi to cpp_extension in build file?

src codes have been chaned to AT, I forgot summit core testing, you can testing them again. best gards!

**Comment by Looottch** [comment]
> > > > > > id you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!
> > > > > 
> > > > > 
> > > > > see DepthAwareStereo
> > > > 
> > > > 
> > > > Your code can't be used in pytorch1.6 right? Cause I try to build ops depthconv from your github page but I got error like "No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda' "
> > > 
> > > 
> > > see testing， training is the same with origin, but tesing has been changed much
> > > I checked testing codes in folder DepthAwareStereo/testing/models/ops/depthconv/src and it's actual original code. build.py is the only different file, will it work when you only change ffi to cpp_extension in build file?
> 
> src codes have been chaned to AT, I forgot summit core testing, you can testing them again. best gards!

sorry to interrupt, I also tried the newest version of your code in the link(models-1804) but it also has problems and  can't be compiled, maybe you missed something?

**Comment by shen1994** [comment]
> > > > > > > id you solve this problem, I'm trying to use depthconv under torch1.2 environment. Can you share some tips with me? Thanks a lot!
> > > > > > 
> > > > > > 
> > > > > > see DepthAwareStereo
> > > > > 
> > > > > 
> > > > > Your code can't be used in pytorch1.6 right? Cause I try to build ops depthconv from your github page but I got error like "No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda' "
> > > > 
> > > > 
> > > > see testing， training is the same with origin, but tesing has been changed much
> > > > I checked testing codes in folder DepthAwareStereo/testing/models/ops/depthconv/src and it's actual original code. build.py is the only different file, will it work when you only change ffi to cpp_extension in build file?
> > 
> > 
> > src codes have been chaned to AT, I forgot summit core testing, you can testing them again. best gards!
> 
> sorry to interrupt, I also tried the newest version of your code in the link(models-1804) but it also has problems and can't be compiled, maybe you missed something?

maybe you need to add depthcov python path to bashrc, so libs can find them
